test/cli/utils_test.go

```diff
@@ -103,15 +103,15 @@ func runCosign(t testing.TB, env map[string]string, args ...string) (*exec.Cmd,
 		env = make(map[string]string)
 	}
 
-	stdout, stderr := runCommand(cmd, env)
+	stdout, stderr := runCommand(t, cmd, env)
 	return cmd, stdout, stderr
 }
 
 func getCosignCommand(t testing.TB, args ...string) *exec.Cmd {
 	return exec.Command(filepath.Join(repoRoot(t), ".tmp/cosign"), args...)
 }
 
-func runCommand(cmd *exec.Cmd, env map[string]string) (string, string) {
+func runCommand(t testing.TB, cmd *exec.Cmd, env map[string]string) (string, string) {
```

Comment:
I think this `t testing.TB` was just added for debugging, and isn't being used? should it be removed again?

Reply:
removed